react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -128,6 +128,11 @@ public void requestImageUploadCancel(final int mediaId) {
         mGutenbergBridgeJS2Parent.requestImageUploadCancel(mediaId);
     }
 
+    @ReactMethod
+    public void nativeLoggingHook(String message, int logLevel) {
```

Comment:
Kinda a nit but, can we perhaps simplify the name of the method to just `log`? The "Hook" part in particular doesn't seem to be that relevant here anyway and I think it's too internal to RN to surface?

Reply:
After a consideration can I leave as it is? Because of the following reasons:

- `log` is a bit generalized name
- by keeping same method name developer can easily figure out the relationship between native and RN by grep

Reply:
(optional)It caught my attention that nativeLoggingHook is a noun but all other bridge methods are imperatives, might be something to improve also.

Reply:
Finally I was convinced😄 a37fcf4...8f1e207